[DJMM] Add troubleshooting section after setup instructions (#26333)
DataDog/documentation: layouts/shortcodes/djm-install-troubleshooting.en.md

> The init script installs the Datadog Agent. To make sure it is properly installed, run the Agent status command:

Reviewer: Can you add details of where and how you can run this? e.g. could just be links to how to ssh into a node on the cluster for the different platforms.

Reviewer 2: Just a heads up, we don't use the FAQ style at Datadog. So let's keep this as a troubleshooting section.

Author: Good to know, thanks a lot !

> The init script installs the Datadog Agent. To make sure it is properly installed, run the Agent status command:

Reviewer: Let's start this as more of a FAQ section. e.g. let's have this content under the question:

I installed Data Jobs Monitoring but don't see any data in the product.
1. The init script installs...

Author: I see, thanks. For now, I believe "I installed Data Jobs Monitoring but don't see any data in the product." is the only question that this troubleshooting section aims to answer. I agree that checking the Agent status is only the first part of the answer, we intended to add a second point on checking the Tracer injection later.

I would be in favor of keeping the current structure for now so that the customer that we know is trying to set up DJM can find some answers quickly, and then plan in the backlog a task to create a more exhaustive FAQ. Is that ok for you ?

I've added a sentence at the top of the section, saying it specifically adresses the lack of data after installation in DJM.

> Data Jobs Monitoring requires to have the Datadog Agent running in the background. You can check that it is correctly installed and running on your cluster with this command:

Reviewer 2: ```suggestion
Data Jobs Monitoring requires a Datadog Agent running in the background. You can check that it is correctly installed and running on your cluster with this command:
```

  ```shell
  sudo datadog-agent status
  ```

> If it is not the case, you may want to check the log file of the installation. On your cluster, these logs are located in `/tmp/datadog-djm-init.log`.

Reviewer 2: ```suggestion
If there is no Agent running, check the log file of the installation. On your cluster, these logs are located in `/tmp/datadog-djm-init.log`.
```

Author: Hi Heston, I'm sorry I had unpushed changes suggesed by my team. I just pushed them, can you take another look please ?

> For further support, make sure the init script contains the following line so that the install logs are sent to the Datadog support team.

Reviewer 2: ```suggestion
For further support, make sure the init script contains the following line to ensure that the install logs are sent to the Datadog support team.
```
